transports/azure-service-bus/legacy/index.md

```diff
@@ -27,6 +27,18 @@ Note: Publish/Subscribe and Timeouts (including message deferral) are supported
 
 include: azure-transports
 
+## Transport at a glance
+
+|Feature                    |   |  
+|:---                       |---
+|Transactions |None, ReceiveOnly, SendsWithAtomicReceive
+|Pub/Sub                    |Native
+|Timeouts                   |Native
+|Large message bodies       |via higher tier (i.ex. Premium) or DataBus
+|Scale-out             |Competing consumer
+|Scripted Deployment        |Supported for the Forwarding Topology using `NServiceBus.Transport.AzureServiceBus.CommandLine`, not supported for the Endpoint Oriented Topology
```

Comment:
It's not supported, but not intended. NServiceBus.Transport.AzureServiceBus.CommandLine is only intended for NServiceBus.Transport.AzureServiceBus transport.

Reply:
@danielmarbach 👆

Reply:
Well but the tool creates all the necessary topology parts of forwarding topology which is independent of the transport. Given that the new transport has to stay compliant with Forwarding topo there is nothing we can break there which means the tool can be used. I get that it was not intended to be used that way but it can that's why I mentioned it here.

Reply:
But given this is the legacy transport and it is transport at a glance I'm happy to drop it

Reply:
☝️ exactly that. We're in the mids of deprecating the legacy ASB transport and having documentation mixing the tool that was designed for the new ASB transport would just add confusion.